common-moonmage.lic

```diff
@@ -127,15 +127,46 @@ module DRCMM
   # Returns false if you're not holding a moon weapon, or you are but can't wear it.
   # https://elanthipedia.play.net/Shape_Moonblade
   def wear_moon_weapon?
-    moon_weapon_regex = /moon[\w]+/
-    moon_wear_messages = ["telekinetic", "already wearing", "You can't wear", "Wear what"]
+    moon_weapon_regex = /((black|red-hot|blue-white) moon[\w]+)/
+    moon_wear_messages = ["You're already", "You can't wear", "Wear what", "telekinetic"]
     wore_it = false
     if DRC.left_hand =~ moon_weapon_regex
-      wore_it = wore_it || bput("wear #{DRC.left_hand}", *moon_wear_messages) =~ /telekinetic/
+      wore_it = wore_it || DRC.bput("wear #{DRC.left_hand}", *moon_wear_messages) == "telekinetic"
     end
     if DRC.right_hand =~ moon_weapon_regex
-      wore_it = wore_it || bput("wear #{DRC.right_hand}", *moon_wear_messages) =~ /telekinetic/
+      wore_it = wore_it || DRC.bput("wear #{DRC.right_hand}", *moon_wear_messages) == "telekinetic"
```

Comment:
When testing this script one-off via `;e echo DRCMM.wear_moon_weapon?` Lich complained about `bput` function didn't exist so I fully qualified the method call.

Additionally, I wanted to ensure this function returned a proper boolean not just a "truthy/falsey" value so changed the `=~` to `==`.